arch/power/compare256_power8.c

```
@@ -0,0 +1,58 @@
+
+    return 256;
+}
+
+Z_INTERNAL uint32_t compare256_power8(const uint8_t *src0, const uint8_t *src1) {
+    return compare256_power8_static(src0, src1);
+}
+
+#define LONGEST_MATCH       longest_match_power8
+#define COMPARE256          compare256_power8_static
+
+#include "match_tpl.h"
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)